website/docs/r/data_factory_integration_runtime_azure.html.markdown

```diff
@@ -46,6 +46,8 @@ The following arguments are supported:
 
 * `description` - (Optional) Integration runtime description.
 
+* `cleanup_enabled` - (Optional) Cluster will not be recycled and it will be used in next data flow activity run until TTL (time to live) is reached if this is set as false. Default is true.
```

Comment:
```suggestion
* `cleanup_enabled` - (Optional) Cluster will not be recycled and it will be used in next data flow activity run until TTL (time to live) is reached if this is set as `false`. Default is `true`.
```